### PR TITLE
copy adapter outputs to staging area

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]
Lgtm

File: setup.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
             'get-upload-urn=pipeline_tools.shared.submission.get_upload_urn:main',
             'get-files-to-upload=pipeline_tools.shared.submission.get_files_to_upload:main',
             'confirm-submission=pipeline_tools.shared.submission.confirm_submission:main',
+            'copy-adapter-outputs=pipeline_tools.shared.submission.copy_adapter_outputs:main'
```
Comment (jessicaway) [inline setup.py:42]: this is the same way I added things to this script, but I am not sure if it worked. 

@wjdingman1 will this allow us to call `copy-adapter-outputs` from the command line, or are there additional changes required?

Comment (wjdingman1) [reply setup.py:42]: yep! setuptools will bundle it and make it an executable available from the command line (provided that the path specified has a  __main__ entrypoint) 

i just built the docker image from that branch and tested it to make sure and it looked good!

Comment (jessicaway) [reply setup.py:42]: Great, thanks!

**Pull request merged**